terraform/README.md

```diff
@@ -11,3 +11,46 @@ The following resources must be created prior to running this Terraform:
  - a persistent storage account in which to store the state must exist
  - a sufficiently-permissioned role must exist and the credentials must be saved
    as secrets on this repository
+
+All subsequent steps are going to assume that these are both configured.
```

Comment:
These steps only need to be ran one time ever right?

Reply:
I mean... sort of? You don't need to do it every time you deploy. But there are valid reasons to create different roles or use different storage accounts.